ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -244,6 +278,86 @@ Rectangle {
         }
     }
 
+    Popup {
+        property var emojis
+        property string shortname
+
+        function openPopup(emojisParam, shortnameParam) {
+            emojis = emojisParam
+            shortname = shortnameParam
+            emojiSuggestions.open()
+        }
+
+        function addEmoji(index) {
+            if (index === undefined) {
+                index = emojiList.currentIndex
+            }
+
+            const message = extrapolateCursorPosition();
+            const unicode = emojiSuggestions.emojis[index].unicode_alternates || emojiSuggestions.emojis[index].unicode
+            replaceWithEmoji(message, emojiSuggestions.shortname, unicode)
+        }
+
+        id: emojiSuggestions
+        width: parent.width - Style.current.padding * 2
+        height: Math.min(400, emojiList.contentHeight + Style.current.smallPadding * 2)
+        x : Style.current.padding / 2
+        y: -height - Style.current.smallPadding
+        background: Rectangle {
+            visible: emojiSuggestions.emojis.length > 0
+            color: Style.current.secondaryBackground
```

Comment:
Should this be consistent with the other popups (stickers, commands, emojis)? It looks a bit off to me: !<img src="https://imgur.com/5lyVMHS.png" width=500>

Reply:
I copied Discord 🤫 
I'd wait until @simonricoo does a design before trying to refactor it